java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -1014,7 +1016,8 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_binaryOpVV(JNIEnv *env,
 
 JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_binaryOpVS(JNIEnv *env, jclass,
                                                                     jlong lhs_view, jlong rhs_ptr,
-                                                                    jint int_op, jint out_dtype) {
+                                                                    jint int_op, jint out_dtype,
+                                                                    jint scale) {
```

Comment:
We need to use the scale not just pass it in.

Reply:
done